### PR TITLE
Work on feedback provided on current UI

### PR DESCRIPTION
**Pull request by esirK**
## Description
The following PR adds the following updates
- Suppress the 'save', and ‘save and add another’ buttons from all tabs except for the last tab
- Make 'save and continue editing' action to only move onto the next tab instead of trying to save
- Use a general title **Add Article** for all Add actions, as opposed to more specific **, Add Newspaper**, etc 
- Only show menu items for which the user has permissions to see
-  Remove default answer for COVID related story
## Type of change
- [x] New feature (non-breaking change which adds functionality)

## Screenshots
**Suppress the 'save', and ‘save and add another’ buttons from all tabs except for the last tab**
Screenshot 2020-08-31 at 15 31 21
Screenshot 2020-08-31 at 15 32 16
**Only show menu items for which the user has permissions to see Current user has no permissions to view Users or Groups
Screenshot 2020-08-31 at 15 36 49
**
Screenshot 2020-08-31 at 15 34 52

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: gmmp/locale/es/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -208,6 +208,14 @@ msgstr ""
 msgid "Welcome,"
 msgstr ""
 
+#: gmmp/templates/admin/change_form.html:6
+msgid "Home"
+msgstr ""
```
Comment (kilemensi) [inline gmmp/locale/es/LC_MESSAGES/django.po]: Missing translation

File: gmmp/locale/fr/LC_MESSAGES/django.po [hunk 2]
```diff
@@ -209,6 +209,14 @@ msgstr ""
 msgid "Welcome,"
 msgstr ""
 
+#: gmmp/templates/admin/change_form.html:6
+msgid "Home"
+msgstr ""
```
Comment (kilemensi) [inline gmmp/locale/fr/LC_MESSAGES/django.po]: Missing translation

**Review by kilemensi — COMMENTED** [review]
👍 @esirK 

I think lets address the stuff we discussed in the review call & we're good to go

1. Rename the button to `Next` or even `Next Tab`,
2. Don't show the `Save and continue editing`, just `Save` and `Save and add another`.

--------------

Lets create a story for making sure if yes is selected in optional question then we validate that an answer is actually given (unless you can also do that in this PR).

**Comment by esirK** [comment]
> 👍 @esirK
> 
> I think let's address the stuff we discussed in the review call & we're good to go
> 
> 1. Rename the button to `Next` or even `Next Tab`,
> 2. Don't show the `Save and continue editing`, just `Save` and `Save and add another`.
> 
> Let's create a story for making sure if yes is selected in optional question then we validate that an answer is actually given (unless you can also do that in this PR).

I have created the appropriate story here https://www.pivotaltracker.com/story/show/174639229. 
Also, we can enquire if we should let users move onto the next tab if they haven't filled in the answers in the current tab.

**Pull request merged**